### PR TITLE
Disable 3 tests that frequently fail.

### PR DESCRIPTION
**Pull request by MathieuDutSik**
## Motivation

Some tests in the CI are frequently failing for largely unknown reasons. It makes sense to disable some of them for the ease of the developer and to isolate what goes wrong.

## Proposal

The following is done:
* The test `test_wasm_end_to_end_non_fungible` is disabled as it is affected by the error `Error: Could not find application URI: .... after 15 tries`.
* The test `test_wasm_end_to_end_fungible` is disabled as it is affected by the error `Error: Could not find application URI: .... after 15 tries`.
* The test `test_end_to_end_reconfiguration::scylladb_grpc` is disabled as it is affected by timeouts.

## Test Plan

We are testing less things.

## Release Plan

Not relevant.

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

**Review by afck — APPROVED** [review]
Do we have a ticket to fix and re-enable them?

File: linera-service/tests/end_to_end_tests.rs [hunk 1]
```diff
@@ -612,6 +612,7 @@ async fn test_wasm_end_to_end_social_user_pub_sub(config: impl LineraNetConfig)
     Ok(())
 }
 
+#[ignore]
```
Comment (afck) [inline linera-service/tests/end_to_end_tests.rs:615]: Are all three tests failing in CI with all configurations? If it's e.g. only ScyllaDB, we should ignore only that test case instead (or maybe just comment it out).

Comment (MathieuDutSik) [reply linera-service/tests/end_to_end_tests.rs:615]: Correct, only the scylladb one has some issues.

**Comment by MathieuDutSik** [comment]
> Do we have a ticket to fix and re-enable them?

Yes, now done

File: linera-service/tests/end_to_end_tests.rs (hunk 1 above)
Comment (MathieuDutSik) [reply linera-service/tests/end_to_end_tests.rs]: I indicated more clearly what tests are failing which should help for the debugging.

File: linera-service/tests/end_to_end_tests.rs [hunk 2]
```diff
@@ -881,6 +883,10 @@ async fn test_wasm_end_to_end_same_wallet_fungible(
     Ok(())
 }
 
+// TODO(#2051): The test `test_wasm_end_to_end_non_fungible::service_grpc` is frequently failing
+// with the error `Error: Could not find application URI: .... after 15 tries`.
+// The whole test is marked as ignored since `tokio::test` needs at least one test.
```
Comment (afck) [inline linera-service/tests/end_to_end_tests.rs:888]: Can't we annotate the test with something like
```rust
#[cfg(any(feature = "scylladb", feature = "dynamodb", feature = "kubernetes", feature = "remote_net"))]
```
instead?
(Or maybe you have to nest three `any`s for that.)

Comment (MathieuDutSik) [reply linera-service/tests/end_to_end_tests.rs]: Yes, that could be a solution. I admit I was a little annoyed by this. I thought that the `tokio::test` should handle the case of zero test matching.

**Pull request merged**